bookwyrm/models/book.py

```diff
@@ -363,6 +365,16 @@ def save(self, *args, **kwargs):
             for author_id in self.authors.values_list("id", flat=True):
                 cache.delete(f"author-books-{author_id}")
 
+        # Create sort title by removing articles from title
+        if self.sort_title is None:
+            articles = chain(
+                *(LANGUAGE_ARTICLES[language] for language in self.languages)
```

Comment:
You can use `LANGUAGE_ARTICLES.get(language)` here to avoid throwing an error when the language isn't known